### PR TITLE
FIX Twitch Provider in 1.0

### PR DESCRIPTION
**Pull request by zAfLu**
See #242

### CODE REVIEW

File: src/AspNet.Security.OAuth.Twitch/AspNet.Security.OAuth.Twitch.csproj [hunk 1]
```diff
@@ -16,6 +16,10 @@
     <Compile Include="..\..\shared\AspNet.Security.OAuth.Extensions\*.cs" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (kinosang) [inline src/AspNet.Security.OAuth.Twitch/AspNet.Security.OAuth.Twitch.csproj]: remove this, keep the csproj file clear

Comment (kinosang) [inline src/AspNet.Security.OAuth.Twitch/AspNet.Security.OAuth.Twitch.csproj]: remove this, keep the csproj file clear

**Review by kinosang — CHANGES_REQUESTED** [review]

File: src/AspNet.Security.OAuth.Twitch/AspNet.Security.OAuth.Twitch.csproj (hunk 1 above)
Comment (zAfLu) [reply src/AspNet.Security.OAuth.Twitch/AspNet.Security.OAuth.Twitch.csproj]: Should be removed now

**Comment by kevinchalet** [comment]
Merged, thanks for your contribution :+1:

**Pull request closed without merging**